### ISSUE TITLE
[SUGGESTION] Identify Mod Conflicts

### ISSUE THREAD

**Issue by TomShkurti** [thepwrtank18/BlackMesaWorkshopManager#3]
Since VPKs have a list of all of their files inside of them, it might also be useful to have the manager flag conflicting mods that modify the same file. Cool things can be done with this like having a pulldown menu to select "exclusive" mods that all modify a common file like character_manifest.txt or the closed caption files, now that enabling/disabling specific mods programmatically is on the table.

**Comment by thepwrtank18** [comment]
Good idea, don't know how to execute it though. I'll ser what I can do.
